R/helpers.R

```diff
@@ -97,3 +97,15 @@ view_url <- function(..., open = interactive()) {
   }
   invisible(url)
 }
+
+template_file <- function(...) {
```

Comment:
Also, there's `find_template()`, it accepts one argument only but it should be fine.

Reply:
So much similar named functions with the same functionality.